openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -805,15 +805,15 @@ For Cryptographic Key Binding, the Client has the following options defined in (
 
 ## Credential Request {#credential-request}
 
-A Client makes a Credential Request to the Credential Endpoint by sending the following parameters in the entity-body of an HTTP POST request using the `application/json` media type.
+A Client makes a Credential Request to the Credential Endpoint by sending the following parameters in the entity-body of an HTTP POST request.
```

Comment:
While reviewing, I was wondering the while time where the response encryption is described. To me it seems it is in the combination of the section about message encryption and metadata. To improve readability and help the understanding of the readers, I suggest to add text here pointing out credential requests can be encrypted (on top of TLS) and refer to the section about encrypted messages and metadata sections.